airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/CreateConnectorModal.tsx

```diff
@@ -139,32 +143,40 @@ const CreateConnectorModal: React.FC<IProps> = ({ onClose, onSubmit, errorMessag
                           <FormattedMessage id="admin.connectorName" />
                         </Label>
                       }
+                      error={meta.touched && !!meta.error}
+                      message={meta.touched && meta.error && <FormattedMessage id={meta.error} />}
                     />
                   )}
                 </Field>
               </FieldContainer>
               <FieldContainer>
                 <Field name="dockerRepository">
-                  {({ field }: FieldProps<string>) => (
+                  {({ field, meta }: FieldProps<string>) => (
                     <LabeledInput
                       {...field}
                       type="text"
                       autoComplete="off"
                       placeholder={formatMessage({
-                        id: "admin.dockerRepository.placeholder",
+                        id: `${
+                          isCloudApp() ? "admin.dockerFullImageName.placeholder" : "admin.dockerRepository.placeholder"
```

Comment:
I thought more about this, and I couldn't think of a better solution using `FeatureService`. Similar to the [OSS/cloud specific markdown rendering](https://github.com/airbytehq/airbyte/blob/master/airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/DocumentationPanel.tsx#L70), I think it's ok to use `isCloudApp()` here.